gapic/schema/api.py

```diff
@@ -474,6 +475,19 @@ def requires_package(self, pkg: Tuple[str, ...]) -> bool:
             for message in proto.all_messages.values()
         )
 
+    def get_custom_operation_service(self, method: "wrappers.Method") -> "wrappers.Service":
```

Comment:
Looks like this touches only one annotation among any (the operation_service) one. What about the others?

Reply:
The important ones are already expose, this one was just a special case.